naga/src/back/spv/block.rs

```diff
@@ -2039,14 +2063,33 @@ impl<'w> BlockContext<'w> {
         }
     }
 
+    /// Generate one or more SPIR-V blocks for `naga_block`.
+    ///
+    /// Use `label_id` as the label for the SPIR-V entry point block.
+    ///
+    /// If control reaches the end of the SPIR-V block, terminate it according
+    /// to `exit`.
+    ///
+    /// If the block contains [`Break`] or [`Continue`] statements,
+    /// `loop_context` supplies the labels of the SPIR-V blocks to jump to. If
+    /// either of these labels are `None`, then it should have been a Naga
+    /// validation error for the corresponding statement to occur in this
+    /// context.
+    ///
+    /// Return `true` if control can reach the end of the block. This means that
+    /// `exit` was used. Otherwise, return `false`, indicating that `exit` was
+    /// unused.
```

Comment:
nitpick: This last paragraph no longer applies, now that we've refactored to use `BlockExitDisposition`. I think we can just remove it?

Pushed suggestion as a `fixup!` with 1505975d.